src/backends/caffe2/caffe2model.cc

```diff
@@ -25,8 +25,9 @@
 
 namespace dd {
 
-  Caffe2Model::Caffe2Model(const APIData &ad)
-    :MLModel()
+  Caffe2Model::Caffe2Model(const APIData &ad, APIData &adg,
+			   const std::shared_ptr<spdlog::logger> &logger)
+    :MLModel(ad, adg, logger)
```

Comment:
Now that MLModels are created with a logger, could a reference be keeped so
```read_from_repository(const std::shared_ptr<spdlog::logger> &logger)```
and
```update_from_repository(const std::shared_ptr<spdlog::logger> &logger)```
can use it ?

Reply:
Good point! will look at it.

Reply:
Indeed, the context are different, and so should be the loggers.
But maybe we should at least change the constructors, so the call
```read_from_repository(ad.get("repository").get<std::string>(), spdlog::get("api"))```
becomes
```read_from_repository(ad.get("repository").get<std::string>(), logger)```